src/utils/tweetUpdates.ts

```diff
@@ -0,0 +1,89 @@
+import { wikiActivities } from '../services/types/activityResult'
```

Comment:
everything in this bot is under utils? :P we could think of a better structure

Reply:
😅 true, i will search for a better way to add the logic